app/src/main/res/values/strings.xml

```diff
@@ -367,71 +367,71 @@
+    <string name="scanning_is_still_in_progress">Scanning still in progress</string>
     <plurals name="no_of_trackers">
         <item quantity="one">%d tracker</item>
         <item quantity="other">%d trackers</item>
     </plurals>
     <string name="install">Install</string>
     <string name="update">Update</string>
     <string name="source_code">Source Code</string>
-    <string name="source_code_links" translatable="false">Github: https://github.com/MuntashirAkon/AppManager
-        \nGitlab: https://gitlab.com/muntashir/AppManager
-        \nRiseup: https://0xacab.org/muntashir/AppManager
+    <string name="source_code_links" translatable="false">GitHub: https://github.com/MuntashirAkon/AppManager
+        \nGitLab: https://gitlab.com/muntashir/AppManager
+        \nRiseUp: https://0xacab.org/muntashir/AppManager
         \nCodeberg: https://codeberg.org/muntashir/AppManager</string>
     <string name="community">Telegram Community</string>
     <string name="community_links">Update Channel: https://t.me/AppManagerChannel
         \nSupport Group: https://t.me/AppManagerAndroid</string>
     <string name="pref_import_existing">Import existing rules</string>
     <string name="pref_import_existing_msg">Add components disabled by others apps to App Manager.
-        Be careful when using this tool as there can be many false positives. Choose only the apps that you\'re certain about</string>
-    <string name="failed_to_get_data_usage_information">Failed to get data usage information</string>
+        Be careful when using this tool as there can be many false positives. Choose only the apps you\'re certain about</string>
+    <string name="failed_to_get_data_usage_information">Could not fetch data usage info</string>
     <string name="clear_cache">Clear Cache</string>
     <string name="block_unblock_trackers_description">Block or unblock ad and tracking components from all the installed apps</string>
     <string name="block_components_dots">Block components…</string>
     <string name="block_components_description">Block all components identified by the given signature(s)</string>
     <string name="deny_app_ops_dots">Deny app ops…</string>
-    <string name="deny_app_ops_description">Deny given app operations identified by the constant values, ie. 63 means RUN_IN_BACKGROUND</string>
+    <string name="deny_app_ops_description">Deny given app operations identified by the constant values, i.e. 63 means RUN_IN_BACKGROUND</string>
     <string name="clear_data_from_uninstalled_apps">Clear data from uninstalled apps</string>
     <string name="clear_data_from_uninstalled_apps_description">Clear data from apps that are uninstalled with DONT_DELETE_DATA flag</string>
     <string name="clear_app_cache">Clear app cache</string>
-    <string name="clear_app_cache_description">Clear cache from all the applications</string>
+    <string name="clear_app_cache_description">Clear app cache from all apps</string>
```

Comment:
**App** is used twice. **Clear cache from all apps** would be more concise without any loss of meaning.